src/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import { ClassNames } from '@emotion/core';
```

Comment:
```suggestion
/* @jsx jsx */
import { ClassNames, jsx } from '@emotion/core';
```

Reply:
Is this required even though I'm not using any special feature of emotion's `jsx?`

Reply:
In general, it's not required in your particular case right now, but I'd add it, as we do that in the majority of our components.
Anyway, I won't complain if you leave it as is.
I believe emotion's jsx is needed only if you do the following `<span css={styles}>a</span>`